spec/form_models/job_details_form_spec.rb

```diff
@@ -82,14 +82,16 @@
     it "a JobDetailsForm can be converted to a vacancy" do
```

Comment:
Can we add a spec in this file for the validations (contract type required, and contract type duration when fixed term) please?